targets/nuttx-stm32f4/Makefile

```diff
@@ -54,7 +54,7 @@ updateobjs:
 cleanlibs: updateobjs
 	rm -f $(OBJS)
 
-clean: cleanlibs
+clean:: cleanlibs
```

Comment:
Is it intentional? This looks accidental to me.

Reply:
No, this is intentional. NuttX [makefiles](https://bitbucket.org/nuttx/apps/src/b52aed887817a3a6374742e9d5d6298ebe300b08/Application.mk?at=nuttx-7.27&fileviewer=file-view-default#Application.mk-250) have changed and the `clean` target became a [double-colon rule](https://www.gnu.org/software/make/manual/make.html#Double_002dColon). From the docs:

> When a target appears in multiple rules, all the rules must be the same type: all ordinary, or all double-colon.

So, I had to update the `clean` rule here, too.